docs/serving/autoscaling/scale-bounds.md

```diff
@@ -48,6 +48,9 @@ spec:
 This value controls the maximum number of replicas that each revision should have.
 Knative will attempt to never have more than this number of replicas running, or in the process of being created, at any one point in time.
 
+If global key `max-scale-limit` is set, both global max scale and per-revision max scale annotation for new revisions will be verified to not exceed it.
```

Comment:
```suggestion
If the global key `max-scale-limit` is set, both the global max scale and per-revision max scale annotation for new revisions will be verified to not exceed it.
```

Reply:
Not a huge fix - will prob be ok without it but I think it reads a little better with the update. The sentence below this could still prob use an update and maybe something other than disallowed - but we prob need input from a real technical writer instead of me.

Reply:
sounds good. thanks! i'll tag @abrennan89 for review.

Reply:
I'd go one further and say:

"If the `max-scale-limit` global key is set, Knative ensures that neither the global max scale nor the per-revision max scale for new revisions exceed this value."

Just because it's not clear otherwise what "verifies" what, IMHO :slightly_smiling_face: lmk what you think though @taragu and when this is ready I can merge it. Thanks.